examples/ui-demo/src/app/providers.tsx

```diff
@@ -16,11 +16,22 @@ const alchemyConfig = createConfig(
     chain: arbitrumSepolia,
     ssr: true,
     policyId: process.env.NEXT_PUBLIC_PAYMASTER_POLICY_ID,
+    enablePopupOauth: true,
   },
   {
     illustrationStyle: DEFAULT_CONFIG.ui.illustrationStyle,
     auth: {
-      sections: [[{ type: "email" as const }], [{ type: "passkey" as const }]],
+      sections: [
+        [{ type: "email" as const }],
+        [{ type: "passkey" as const }],
+        [
+          {
+            type: "social" as const,
+            authProviderId: "google",
+            mode: "popup",
+          },
+        ],
+      ],
```

Comment:
ah yea, so if you want these in the same section, you can remove the sections check above and do this instead:

```suggestion
        [{ type: "passkey" as const }, {
            type: "social" as const,
            authProviderId: "google",
            mode: "popup",
          }],
      ],
```